### DISCUSSION TITLE
How to show rectangle boxes around cell?

### DISCUSSION THREAD

**Discussion by thezjy** [Dyalog/ride#761] [Q&A] [answered]
I download the free version of Dyalog 18. When I type expressions in the editor, the printed values are not surrounded by rectangle boxes like those on tryapl.org. How can I turn it on?

**Comment by abrudz** [comment, marked as answer]
Not a RIDE issue. Simply enter:
```
]box on
```
For further tips and tricks like this one, check out https://dyalog.com/getting-started/tips.htm

**Reply by thezjy** [reply]
Thanks for your super fast reply! Just found it here https://www.dyalog.com/tools/user-commands.htm.
